package.json

```diff
@@ -20,9 +20,15 @@
     "build": "grunt dist",
     "change": "grunt chg-add",
     "clean": "grunt clean",
+    "grunt": "grunt",
```

Comment:
Most of this will get removed before it's ready. Makes working with this locally a bit easier.
It'll probably get pulled into a separate PR.